OADP-4074-network-settings (#82622)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.adoc

:_mod-docs-content-type: ASSEMBLY
[id="oadp-recommended-network-settings.adoc"]
= {oadp-short} recommended network settings
include::_attributes/common-attributes.adoc[]
include::_attributes/attributes-openshift-dedicated.adoc[]

:context: oadp-recommended-network-settings

toc::[]

For a supported experience with {oadp-first}, you should have a stable and resilient network across {OCP-short} nodes, S3 storage, and in supported cloud environments.

To ensure successful backup and restore operations, Red Hat recommends that your network settings meet the following minimum requirements:

> * Bandwidth: Greater than 2 MB

Reviewer: Maybe also add something like, `Greater than 2 Mbps for small backups, but consider higher bandwidth (10-100 Mbps) based on the data volume`

Author: Incorporated.

> * Bandwidth: Greater than 2 MB

Author: Yes, after incorporating both yours and Shubham's suggestions, I've finalized the wording:

- Bandwidth (network upload speed to object storage): Greater than 2 Mbps for small backups and 10-100 Mbps depending on the data volume for larger backups.

* Packet loss: Range 1% to 5%
* Packet corruption: Range 1% to 5%
* Latency: Range 100ms (±20ms)

> 

Reviewer: Lets also add about Jitter, I suppose we have some data regarding that as well

Author: @shubham-pampattiwar Since jitter refers to the the variation in the interval between successive data packets, should we include it as just a note, or add a recommended range for it as well?
FYI @weshayutin and @TzahiAshkenazi

Author: Incorporated based on Tzahi's inputs.

:_mod-docs-content-type: ASSEMBLY
[id="oadp-recommended-network-settings.adoc"]
= {oadp-short} recommended network settings
include::_attributes/common-attributes.adoc[]
include::_attributes/attributes-openshift-dedicated.adoc[]

:context: oadp-recommended-network-settings

toc::[]

For a supported experience with {oadp-first}, you should have a stable and resilient network across {OCP-short} nodes, S3 storage, and in supported cloud environments that meet {OCP-short} network requirement recommendations.

> To ensure successful backup and restore operations for deployments with remote S3 buckets located off-cluster with suboptimal data paths, Red Hat recommends that your network settings meet the following minimum requirements in such less optimal conditions:

Reviewer 2: I think we prefer to use _it is recommended_ in such situations (as awkward as it sounds to me).
```suggestion
To ensure successful backup and restore operations for deployments with remote S3 buckets located off-cluster with suboptimal data paths, it is recommended that your network settings meet the following minimum requirements in such less optimal conditions:
```

* Bandwidth (network upload speed to object storage): Greater than 2 Mbps for small backups and 10-100 Mbps depending on the data volume for larger backups.
* Packet loss: 1%
* Packet corruption: 1%
* Latency: 100ms

> Ensure that your OCP network performs optimally and meets OpenShift network requirements.

Reviewer 2: ```suggestion
Ensure that your {product-title} network performs optimally and meets {product-title} network requirements.
```

[IMPORTANT]
====

> While Red Hat supports standard backup and restore failures, it does not support failures caused by network settings that do not meet the recommended thresholds.

Reviewer 2: Per IBM Style Guide, use _while_ only to refer to time. 
```suggestion
Although Red Hat supports standard backup and restore failures, it does not support failures caused by network settings that do not meet the recommended thresholds.
```
Also, I wonder if _Red Hat supports_ could be worded differently? (I don't think RH supports failures; we don't have much of choice, do we?)
```suggestion
Although Red Hat provides supports for standard backup and restore failures, it does not provide support for failures caused by network settings that do not meet the recommended thresholds.
```
